interfaces/Portalicious/src/app/pages/change-password/change-password.page.html

```diff
@@ -3,49 +3,6 @@
   i18n-pageTitle="@@page-title-change-password"
 >
   <p-card styleClass="max-w-xl">
-    <app-form-default
```

Comment:
Where is the code for the `ChangePasswordComponent`?
Did you forget to commit it, or is it already somewhere else in the codebase?